sympy/printing/mathml.py

```diff
@@ -1221,6 +1221,38 @@ def _print_EmptySet(self, e):
         return x
 
 
+    def _print_floor(self, e):
+        mrow =  self.dom.createElement('mrow')
+        x = self.dom.createElement('mfenced')
+        x.setAttribute('open', u'\u230A')
+        x.setAttribute('close', u'\u230B')
+        x.appendChild(self._print(e.args[0]))
+        mrow.appendChild(x)
+        return mrow
+
+
+    def _print_ceiling(self, e):
+        mrow =  self.dom.createElement('mrow')
+        x = self.dom.createElement('mfenced')
+        x.setAttribute('open', u'\u2308')
+        x.setAttribute('close', u'\u2309')
+        x.appendChild(self._print(e.args[0]))
+        mrow.appendChild(x)
+        return mrow
+
+
+    def _print_Lambda(self, e):
+        x = self.dom.createElement('mfenced')
+        mrow = self.dom.createElement('mrow')
+        mrow.appendChild(self._print(e.args[0]))
```

Comment:
One could update the code similar to the LaTeX code that if there is only one symbol print it without the brackets:
```
        symbols = expr.args[0]
        if len(symbols) == 1:
            symbols = self._print(symbols[0])
        else:
            symbols = self._print(symbols)
        mrow.appendChild(symbols)
```